client/js/components/PrivacyPolicyExpander/PrivacyPolicyExpander.js

```diff
@@ -0,0 +1,46 @@
+import React, {Component} from 'react';
+import {View, Text, TouchableOpacity} from 'react-native';
+import PropTypes from 'prop-types';
+import styles from './styles';
+
+export default class PrivacyPolicyExpander extends Component {
+  constructor(props) {
+    super(props);
+  }
+  state = {
+    isCollapsed: true,
+  };
+  onPress() {
+    this.setState({isCollapsed: !this.state.isCollapsed});
+  }
+  render() {
+    const {isCollapsed} = this.state;
+    const {title, description} = this.props;
+    return (
+      <>
+        <TouchableOpacity onPress={() => this.onPress()}>
+          <View style={styles.expandableButton}>
+            <Text style={styles.expandableTitle}>{title}</Text>
+            {!isCollapsed ? (
```

Comment:
you can put string = isCollapsed ? a : b (before the return), then use the string in your component. repeat as few things as possible